easybuild/tools/github.py

```diff
@@ -324,6 +325,20 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
         # make sure path exists, create it if necessary
         mkdir(path, parents=True)
 
+    develop = download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch='develop', path=path)
+    if not os.path.isdir(develop):
+        raise EasyBuildError('Downloading of develop branch failed: not found in %s', develop)
+
+    patch_name = 'patch%s' % pr
+    patch_url = URL_SEPARATOR.join([GITHUB_URL, GITHUB_EB_MAIN, GITHUB_EASYCONFIGS_REPO, GITHUB_PR, str(pr)])
+    patch = download_file(patch_name, patch_url + '.patch', path + '/' + patch_name)
+    if patch is None:
+        raise EasyBuildError('Failed to download file %s to %s', patch_url + '.patch', path + '/' + patch_name)
```

Comment:
don't copy-paste `patch_url + '.patch'` and `path + '/' + patch_name`, use variables:

```python
patch_url = URL_SEPARATOR.join([GITHUB_URL, GITHUB_EB_MAIN, GITHUB_EASYCONFIGS_REPO, GITHUB_PR, '%s.patch' % pr])
patch_path = os.path.join(path, patch_name)
patch = download_file(patch_name, patch_url, patch_path)
if patch is None:
    raise EasyBuildError("Failed to download file %s to %s", patch_url, patch_path)
```